### ISSUE TITLE
Notifications are being sent numerous times

### ISSUE THREAD

**Issue by WillBishop** [christianselig/apollo-bugs#36]
**Describe the bug**
I received two comments within quick succession and now I’ve received a notification for one of them about 6 times, and for the other about 10 times.

**To Reproduce**
Not really sure 

**iOS version**: 12.3 Beta

**Apollo version**: Latest App Store Version (I’m writing this from an SFSafariViewController in Apollo so I can’t check the version, but I’m 99% confident it’s the latest)

**iOS device**: iPhone XS

**Jailbroken**: i wish, but nah

**Reproducible:** Random

**Expected behavior**
One notification per comment

**Comment by christianselig** [comment]
Investigating

**Comment by jeop10** [comment]
This issue have been addressed here for anyone looking for more information:

https://www.reddit.com/r/apolloapp/comments/bqjznz/if_you_were_having_some_bugs_with_notifications/

TLDR: It have been resolved but if you encounter any issue please report it again
